python/cugraph/__init__.py

```diff
@@ -28,7 +28,7 @@
 from cugraph.link_prediction import jaccard, overlap, jaccard_w, overlap_w
 from cugraph.structure import Graph, from_cudf_edgelist, renumber, symmetrize, symmetrize_df
 from cugraph.traversal import bfs, sssp, filter_unreachable
-from cugraph.utilities import grmat_gen
+# from cugraph.utilities import grmat_gen
```

Comment:
I remember us discussing this approach (just comment out all places where this is imported) vs. removing the (temporarily) broken function from the cython interface. This way is easier, but it allows someone to still import `grmat_gen` and crash.  Would it be better to remove the API from cython instead, so graph users cannot import a broken API? Or another option could be to re-write `grmat_gen` in cython to raise `NotImplementedError`?

Reply:
I suppose there's no easy way to remove an API, and either approach will result in a traceback, so I'm fine with it either way. I guess we just have to decide what type of exception we'd want a user to see. @afender ?

Reply:
We're temporarily suspending RMAT. We intend to bring it back in 0.11 (I think). So, possible subsequent bugs that might be opened by users trying to run GRMAT from Cython (which I think is the only way to access it, since direct Python dependencies on it have been removed) will get automatically addressed in next release. Which might be a better scenario than silently removing a feature that people would ask were it went.